javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,37 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
```

Comment:
This is good . everything works fine. 
Optional challenge :
You can use logical operators in the statements to do the the assignment with less lines of codes.